imagebuildah/build.go

```diff
@@ -1370,14 +1401,8 @@ func (s *StageExecutor) commit(ctx context.Context, ib *imagebuilder.Builder, cr
 	if imageRef != nil {
```

Comment:
Is this if statement  useful anymore?

Reply:
At the moment, I think you're right and it can be removed, but #1486 would have to add it back.

Reply:
OK, no worries then, leave it be.